hummingbot/strategy/pure_market_making/pure_market_making_config_map.py

```diff
@@ -174,7 +174,7 @@ def exchange_on_validated(value: str):
                          "additional order? >>> ",
```

Comment:
I think we should change to the prompt message a bit so user knows that decreasing order size is also an option. And we'll need to update doc and the template parameter description as well.

Reply:
Done!